pkgs/default.nix

```diff
@@ -216,8 +216,10 @@ pkgs: oldpkgs: {
         -Wall \
         -x c \
         "$textPath"
-    strip --strip-unneeded "$exe"
-  '';
+    ${optionalString (!pkgs.stdenv.hostPlatform.isDarwin) /* sh */ ''
+      strip --strip-unneeded "$exe"
+    ''}
+    ''};
```

Comment:
One last request, this line needs to become: `  '';`, i.e. remove the `}` and be properly indented.

Reply:
I've taken the liberty to amend your PR with this issue fixed. Hope you don't mind :)

(Beware, I've force-pushed to your master branch!)